### PR TITLE
Add method to run webhook test

### PR DESCRIPTION
**Pull request by jirimoravcik**
Add method to run webhook test;

Should this method be added to the python package? @drobnikj

### CODE REVIEW

**Review by drobnikj — CHANGES_REQUESTED** [review]
Please update the changelog and bump version, otherwise looks good.

**Comment by drobnikj** [comment]
Plus check the linting check, it looks like it failed.

**Comment by drobnikj** [comment]
> Should this method be added to the python package?

If you are about to add it you can, but we can leave it for @fnesveda.

**Review by drobnikj — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+1.2.7 / 2021/07/15
+===================
+- Added new method `.test()` to the `WebhookClient` class
+
```
Comment (mnmkng) [inline CHANGELOG.md:4]: New method is a new feature so it should be a [semver-minor](https://semver.org/#spec-item-7) bump to `1.3.0`. Should've been done in earlier changes as well, but better late than never.

**Review by mnmkng — CHANGES_REQUESTED** [review]

**Review by mnmkng — APPROVED** [review]

**Pull request merged**